src/routes/userRouter.ts

```diff
@@ -1,103 +1,94 @@
-import express, { Request, Response } from 'express'
-import { User } from '../../models/UserModel'
-import jwt from 'jsonwebtoken'
-import bcrypt from 'bcryptjs'
+import express, { Request, Response } from "express";
+import { User } from "@/models/UserModel";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcryptjs";
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/findall', async (req: Request, res: Response) => {
-  const users = await User.find({})
-  return res.status(200).send(users)
-})
+router.get("/findall", async (req: Request, res: Response) => {
+  const users = await User.find({});
+  return res.status(200).send(users);
+});
 
-router.post('/add', async (req: Request, res: Response) => {
+router.post("/add", async (req: Request, res: Response) => {
   // console.log(req.body.name)
-  const check = await User.findOne({name:req.body.name.toLowerCase()})
-  if(!check){
-    const name = req.body.name.toLowerCase();
-    const password=await bcrypt.hash(req.body.password, 10);
-    const user = User.build({ name, password })
-    await user.save()
-    return res.status(201).send(user)
+  const check = await User.findOne({ name: req.body.name.toLowerCase() });
+  if (check) {
+    return res.status(400).send("User exists");
   }
-  else{
-    // console.log(check)
-    return res.status(400).send("User exists")
-    
-  }
-})
+  const name = req.body.name.toLowerCase();
+  const password = await bcrypt.hash(req.body.password, 10);
+  const user = User.build({ name, password });
+  await user.save();
+  return res.status(201).send(user);
+});
 
-router.post('/login', async (req: Request, res: Response) => {
+router.post("/login", async (req: Request, res: Response) => {
   // console.log(req.body.name)
-  const check = await User.findOne({name:req.body.name.toLowerCase()})
-  if(!check){
-    return res.status(400).send("User does not exists")
-  }
-  else{
-    const secretkey : any=process.env.SECRET_KEY
```

Comment:
no need to create new variable use process.env.SECRET_KEY directly to jwt.sign()